### PR TITLE
feat: remove access keys and check txHash from wallet redirect

### PR DESCRIPTION
**Pull request by paouvrard**
- New transaction signing flow for https://github.com/near/rainbow-bridge-frontend/issues/98 and https://github.com/near/rainbow-bridge-frontend/issues/106
- ft-141
- checkStatus error handling

### CODE REVIEW

File: packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js [hunk 1]
```diff
@@ -268,10 +268,56 @@ export async function checkWithdraw (transfer) {
...
+  const successReceiptOutcome = withdrawTx.receipts_outcome
+    .find(r => r.id === txReceiptId).outcome
+  const successReceiptId = successReceiptOutcome.status.SuccessReceiptId
+  const successReceiptExecutorId = successReceiptOutcome.executor_id
+
+  let withdrawReceiptId
+
+  // TODO: refactor withdrawReceiptId into function to be shared by recover()
+  // in separate PR.
+  // Check if this tx was made from a 2fa
```
Comment (chadoh) [inline packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js:280]: Can remove this comment?

**Review by chadoh — APPROVED** [review]
I like the shape of the code. It's as followable as transaction-parsing code can be. 

I did not try it out and am relying on you to ensure it works well.

**Review by chadoh — APPROVED** [review]
Updates since I last reviewed still look good to me. Again, I haven't run the app locally to verify that this catches all edge cases. 

I'm looking forward to [completedTransactions](https://github.com/near/near-api-js/pull/467) being implemented in near-api-js so the urlParams logic can get cleaned up. This logic currently seems very error-prone to me.

File: packages/nep141-erc20/src/natural-erc20/sendToNear/index.js [hunk 2]
```diff
@@ -128,7 +129,8 @@ export async function initiate ({
     completedConfirmations: 0,
     lockHashes: [],
     lockReceipts: [],
-    neededConfirmations: 30 // hard-coding until connector contract is updated with this information
+    neededConfirmations: 30, // hard-coding until connector contract is updated with this information
```
Comment (djsatok) [inline packages/nep141-erc20/src/natural-erc20/sendToNear/index.js:132]: The confirmations are dependant on the bridge instance. For mainnet bridge we use 20 confirmations at the moment, less than that for Ropsten. We can change this into 20 as a quick hack.

**Review by djsatok — APPROVED** [review]

**Pull request merged**